src/Model/AbstractArticle.php

```diff
@@ -58,17 +58,17 @@
     /**
      * @var string
      */
-    protected $title;
+    protected $title = '';
```

Comment:
Be aware, that doctrine will override the default values when loading an entity from the database.

Reply:
Good point, I'll remove this and handle optionnal values where appropriate.

Reply:
@core23 @greg0ire PR updated, could you have a look please ?